packages/compiler/core/types.ts

```diff
@@ -144,7 +144,7 @@ export type IntrinsicModel<T extends IntrinsicModelName = IntrinsicModelName> =
 export interface ModelType extends BaseType, DecoratedType, TemplatedType {
   kind: "Model";
   name: IntrinsicModelName | string;
-  node:
+  node?:
```

Comment:
when do you not have a node?

Reply:
When I create a new type in filterModelProperties. I wasn't sure about this. I could maybe copy the node from the source model, but I recently found issues with places where we do that elsewhere. See https://github.com/microsoft/cadl/issues/462. This will be a good thing to cover in design discussion.

Reply:
right I guess that make sense. I think then we might want to think of a sourceMap system so you can figure out where a type like that was introduced from